### PR TITLE
Applications list

### PR DESCRIPTION
**Pull request by jaleszek**
- applications admin list first sketch
- comments functionality

Some of the methods are hardcoded actually - need to clarify some design decision - it gives option to work on styling this part of app.

### CODE REVIEW

File: app/views/admin/form_answers/index.html.slim [hunk 1]
```diff
@@ -4,28 +4,29 @@ header.page-header.group
       ' Applications
 
 .article-related-positioning-container
-  .article-container
+  .article-container style= 'width: 95%; max-width: 100%;'
```
Comment (RuslanHamidullin) [inline app/views/admin/form_answers/index.html.slim]: minor thing: would be great to move inline style attribute to related css file

**Comment by RuslanHamidullin** [comment]
Looks good for me :+1:

File: app/views/admin/comments/index.html.slim [hunk 2]
```diff
@@ -0,0 +1,2 @@
+h1 FormAnswer::Comments#index
+p Find me in app/views/form_answer/comments/index.html.slim
```
Comment (dreamfall) [inline app/views/admin/comments/index.html.slim]: please, remove this view

File: app/views/admin/comments/show.html.slim [hunk 3]
```diff
@@ -0,0 +1,2 @@
+h1 FormAnswer::Comments#show
+p Find me in app/views/form_answer/comments/show.html.slim
```
Comment (dreamfall) [inline app/views/admin/comments/show.html.slim]: Please, remove this one as well.

File: app/views/admin/form_answers/index.html.slim (hunk 1 above)
Comment (jaleszek) [inline app/views/admin/form_answers/index.html.slim]: The styles for this list are not implemented yet so it's better to just keep it here.

Comment (RuslanHamidullin) [inline app/views/admin/form_answers/index.html.slim]: ok

**Pull request merged**